server/pulp/server/webservices/urls.py

```diff
@@ -31,11 +31,11 @@
                                                            ConsumerGroupUnassociateActionView,
                                                            ConsumerGroupView)
 from pulp.server.webservices.views.content import (
-    CatalogResourceView, ContentSourceView, ContentSourceResourceView,
-    ContentUnitResourceView, ContentUnitSearch, ContentUnitsCollectionView,
-    ContentUnitUserMetadataResourceView, DeleteOrphansActionView, OrphanCollectionView,
-    OrphanResourceView, OrphanTypeSubCollectionView, UploadsCollectionView, UploadResourceView,
-    UploadSegmentResourceView
+    CatalogResourceView, ContentSourceCollectionView, ContentSourceCollectionActionView,
+    ContentSourceResourceView, ContentUnitResourceView, ContentUnitSearch,
+    ContentUnitsCollectionView, ContentUnitUserMetadataResourceView, DeleteOrphansActionView,
+    OrphanCollectionView, OrphanResourceView, OrphanTypeSubCollectionView, UploadsCollectionView,
+    UploadResourceView, UploadSegmentResourceView
 )
 from pulp.server.webservices.views.events import (EventResourceView, EventView)
```

Comment:
These giant paragraph style lists are hard to work with.  This style is much clearer and cleaner:
```
from pulp.server.webservices.views.content import (
    CatalogResourceView,
    ContentSourceView,
    ...
)
```
Just my $0.02.  (but not changing all that now)